content/blogposts/2022/cloud-managed-instances-for-enterprise.md

```diff
@@ -0,0 +1,45 @@
+
+Today — after years of building trust and engineering rigor — we're confidently moving to the Cloud with this highly secure and scalable solution. Dedicated Sourcegraph Cloud instances are now generally available, and the best way for new teams to use Sourcegraph.
+
+## Security and scalability
+
+As a developer, your code is your most valuable IP. This makes security critical for Sourcegraph in all forms.
+
+That’s why we’re offering Sourcegraph Cloud via dedicated, single-tenant instances. When you run Sourcegraph Cloud, even as a trial, you get your own instance dedicated to only **your** code. You get company-level data isolation out-of-the-box.
+
+Plus, Sourcegraph has received a SOC 2 Type II report, so you can be confident that Sourcegraph controls for the security and confidentiality of your sensitive data.
+
+This means you get enterprise-grade scaling. You can sync tens of thousands of repositories to your Sourcegraph instance and search across all of them without slowdowns, so you can stay in flow even when working in huge codebases.
+
+## The latest features in the Cloud
+
+Keeping feature parity between Sourcegraph Self-hosted and Sourcegraph Cloud is another reason we chose a single-tenant architecture. Every Cloud instance is kept up-to-date with the latest Sourcegraph releases.
+
+We also offer our executors technology out-of-the-box on Cloud. Executors are dedicated compute instances outside of the main Sourcegraph service that provide extra compute capacity for specific jobs. Executors are provided by default for every Sourcegraph Cloud instance, so you get access to our latest executor-based features without any configuration.
+
+Executors let you run [server-side Batch Changes](https://docs.sourcegraph.com/batch_changes/explanations/server_side), so you can make broad changes across all your repositories all at once. Batch Changes can be used to fix security vulnerabilities or update old code, and on Cloud, anyone can run large-scale changes without extra configuration or local environment setup. 
+
+Executors also power auto-indexing for code navigation, which lets you index your code continuously so that you always have compiler-accurate, [precise code navigation for your repositories](https://about.sourcegraph.com/blog/announcing-auto-indexing). With auto-indexing, anyone on your team can enable precise code navigation for their own repositories, making it more accessible than ever.
+
+On Cloud, executors require no extra configuration and no infrastructure provisioning. 
+
+## How to get started with Sourcegraph Cloud
+
+Sourcegraph Cloud is the best way to get started with Sourcegraph. If you’d like to try out Sourcegraph’s code intelligence platform for your team, you can [sign up for a trial here](https://www.signup.sourcegraph.com/).
```

Comment:
Signup doesn't exist on a multi level subdomain. Just need to remove the `www.`:

```suggestion
Sourcegraph Cloud is the best way to get started with Sourcegraph. If you’d like to try out Sourcegraph’s code intelligence platform for your team, you can [sign up for a trial here](https://signup.sourcegraph.com/).
```